modules/python.zsh

```diff
@@ -1,19 +1,28 @@
 #!/usr/bin/env zsh
 
+# default is regular 'python'
+_PYTHON=python;
```

Comment:
Rather than using a variable, a function would be preferable:
```
__python_executable_name() {
    echo "python exec name"
}
```

use:
```
command -v $(__python_executable_name) > /dev/null && echo -ne "$($(__python_executable_name) --version 2>&1 | awk '{print $2}' | tr -d ' \n')"
```

Reply:
okay got it. have a look now?